_posts/2024-02-17-stable-release-0.10.6.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Unstable release 0.10.5
+author: Amazinite
+---
+Endless Sky v0.10.6 is now live on Github and Steam! The release will be made live on GOG in the coming days, as all GOG updates must go through a QA test from GOG.
+
+This release is being sent out to everyone, so you don't need to be on the beta branch to get it. This update focuses on bug fixes for the recent v0.10.5 release, so if you haven't played that then take a look at the announcement above or the changelog to see what's new.
+
+A reminder that we have a feedback box for each major release, the most recent one being here. If you haven't looked at them already, it would help to check them out and leave us any feedback.
+
+Also, if you haven't already, head over to Github and star Endless Sky or give us a positive review on Steam or GOG to help the game become more discoverable!
+
+Our next unstable release, v0.10.7, is planned for late May, and is shaping up to be a rather big release in terms of combat balance.
```

Comment:
So no links?
```suggestion
@everyone v0.10.6 is now live on Github and Steam! The release will be made live on GOG in the coming days, as all GOG updates must go through a QA test from GOG.

This release is being sent out to everyone, so you don't need to be on the beta branch to get it. This update focuses on bug fixes for the recent v0.10.5 release, so if you haven't played that then take a look at the announcement above or the [changelog](https://github.com/endless-sky/endless-sky/blob/master/changelog) to see what's new.

A reminder that we have a feedback box for each major release, the most recent one being [here](https://forms.gle/gPERzubizkhyeq6w8). If you haven't looked at them already, it would help to check them out and leave us any feedback.

Also, if you haven't already, head over to [Github](https://github.com/endless-sky/endless-sky) and star Endless Sky or give us a positive review on [Steam](https://store.steampowered.com/app/404410/Endless_Sky/) or [GOG](https://www.gog.com/en/game/endless_sky) to help the game become more discoverable!

Our next unstable release, v0.10.7, is planned for late May, and is shaping up to be a rather big release in terms of combat balance. :eyes:
```

Reply:
Links? I just copy paste.